docs/site/Serving-static-files.md

```diff
@@ -37,11 +37,23 @@ export class TodoListApplication extends BootMixin(
 
     // ...
 
-    this.static('/', path.join(__dirname, '../../public'));
+    this.static('/public', path.join(__dirname, '../../public'));
   }
 }
 ```
 
+If the path prefix overlaps with controller routes (e.g. `'/'`), then you should
+only add the static route _after_ the controllers have been loaded.
+
+```ts
+  async boot() {
+    await super.boot();
+
+    // After all the controllers have loaded, add this fallback route
+    this.static('/', path.join(__dirname, '../../public'));
+  }
+```
+
```

Comment:
This is the solution to the aforementioned issue. It might be simpler if we just recommend this approach by default, and drop the earlier docs which use the constructor. Would you like me to do that?